build.gradle

```diff
@@ -321,7 +321,7 @@ platform {
 //	bundle 'com.ning:async-http-client:1.8.15'
 
 	// quartz scheduler
-	bundle 'org.quartz-scheduler:quartz:1.7.3'
+	bundle 'org.quartz-scheduler:quartz:2.4.0-rc1'
```

Comment:
Any specific reason for using a release candidate version?
Would be great to add that as comment.

Reply:
Good point. Thank you
i was just blind not to see that the 2.3.2, which is just the last version of 2.3 is without vulnerabilities.